src/style/style.spec.ts

```diff
@@ -1,20 +1,25 @@
 import config from "./color-config";
 import mix from "./utils/mix";
-import generatePalette from "./palette";
+import generatePalette, { PaletteFunction } from "./palette";
```

Comment:
Should it be "import { generatePalette, PaletteFunction } from "./palette"; "?

Reply:
The `generatePalette` function is a default export and the `PaletteFunction` is a named so the above syntax is correct